hack/generated/pkg/genruntime/convertible_spec.go

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package genruntime
+
+type ConvertibleSpec interface {
```

Comment:
From talking with you, I understand that the From and To directions are needed because without it there'd need to be circular references between different versions. Can you add a comment to that effect? :grin: